lib/public/Comments/ICommentsManager.php

```diff
@@ -205,7 +205,7 @@ public function getLastCommentBeforeDate(string $objectType, string $objectId, \
 	 * @param string $verb
 	 * @param string $actorType
 	 * @param string[] $actors
-	 * @return array
+	 * @return array Map of "string actor" => "\DateTime most recent comment date"
```

Comment:
```suggestion
	 * @return \DateTime[] Map of "string actor" => "\DateTime most recent comment date"
```

Phpstorm and Psalm don't speak your language :)

Reply:
If you want to be fancy you can add an *additional* `@psalm-return array<string, \DateTime>`